pkilint/bin/lint_pkix_cert.py

```diff
@@ -44,7 +44,11 @@ def main(cli_args=None) -> int:
 
         return 0
     else:
-        cert = loader.load_certificate(args.file, args.file.name)
+        try:
+            cert = loader.load_certificate(args.file, args.file.name)
+        except ValueError as e:
+            print(f'Failed to load certificate: {e}')
```

Comment:
```suggestion
            print(f'Failed to load certificate: {e}', file=sys.stderr)
```

I think writing to stderr provides two benefits in this case:
1. Consistency with the previous behavior of writing unhandled exception messages to stderr
2. Reduces complexity for callers of the CLI tool, as such output from stderr would not be confused with "normal" report outputs

Thoughts?

Reply:
I agree with your reasoning.